src/js/routes/TwitterHandleLanding.jsx

```diff
@@ -151,23 +152,23 @@ export default class TwitterHandleLanding extends Component {
           />
         );
       }
-    } else if (this.state.kind_of_owner === "TWITTER_HANDLE_NOT_FOUND_IN_WE_VOTE") {
+    } else if (this.state.kindOfOwner === "TWITTER_HANDLE_NOT_FOUND_IN_WE_VOTE") {
       // console.log("TwitterHandleLanding TWITTER_HANDLE_NOT_FOUND_IN_WE_VOTE calling UnknownTwitterAccount");
       return <UnknownTwitterAccount {...this.state} />;
```

Comment:
Hello @fulin426, since we are passing this.state into the component UnknownTwitterAccount, you will need to make sure that the incoming props for WebApp/src/js/routes/VoterGuide/UnknownTwitterAccount.jsx get updated to camel case.

Reply:
That is correct @fulin426.